integration-tests/__tests__/custom_reporters.test.js

```diff
@@ -14,7 +14,7 @@ const runJest = require('../runJest');
 const os = require('os');
 const path = require('path');
 
-const DIR = path.resolve(os.tmpdir(), 'custom_reporters_test_dir');
+const DIR = path.resolve(os.tmpdir(), 'custom-reporters_test_dir');
```

Comment:
`custom-reporters-test-dir`

Reply:
Ah yes. Sorry about that. Both of these are fixed.